mutt_curses.h

```diff
@@ -111,6 +111,7 @@ struct KeyEvent
   int op; /**< function op */
 };
 
+void mutt_flush_stdin(void);
```

Comment:
As there's only one use, I think we should drop this and move the function before `mutt_any_key_to_continue()` so that we don't need to forward-declare it.

Reply:
OK